review.php

```diff
@@ -240,7 +240,7 @@ public function is_time() {
 
 			$installed = (int) get_option( $this->key, false );
 
-			if ( false === $installed ) {
+			if ( 0 === $installed ) {
```

Comment:
That's a very good point! Comparing to `0` works for me, but, in this case, we should also set the default value of `get_option()` to `0` as well instead of `false`. Better prepare exactly for what we need instead of relying on a typecast.

```
$installed = (int) get_option( $this->key, 0 );
```

Reply:
Hi - it's me - apparently I was terrible at following up with this! Sorry about that.
Updated now.